routes/trackRouter.js

```diff
@@ -1092,7 +1093,7 @@ router.get('/', function (req, res, next) {
 });
 
 // REQUIRES: req.body to contain a list of track URI's in format ["spotify:track:1ue7zm5TVVvmoQV8lK6K2H", ...]
-router.post('/', function (req, res, next) {
+router.post('/', trackLimiter, function (req, res, next) {
```

Comment:
Cool! Do we just have to attach it to the request and it'll automatically apply the limiter?

Reply:
Yup, and it attaches this to each request that we apply the rate limiter. 
![image](https://user-images.githubusercontent.com/49849754/94350116-df84ff00-ffff-11ea-86c4-870732d3c84f.png)